src/util/AutoEditor.ts

```diff
@@ -38,6 +39,7 @@ export class AutoEditor {
 
     getEdits(): Subject<PadEdit> { return this.edits; }
     getMode(): BehaviorSubject<EditorMode> { return this.mode; }
+    getKeymap(): BehaviorSubject<string> { return this.keymap; }
```

Comment:
does the keymap need to be integrated into the auto editor?  it's not like users will be typing into it.